src/pmrep/pmrep.conf

```diff
@@ -609,7 +609,22 @@ network.interface.out.fifo    = txfifo/s,,,,
 
 #[sar-n-NFS]
 #[sar-n-NFSD]
-#[sar-n-SOCK]
+
+[sar-n-SOCK]
+header = yes
+unitinfo = no
+globals = no
+timestamp = yes
+width = 9
+precision = 2
+delimiter = " "
+network.sockstat.total        = totsck,,,,
+network.sockstat.tcp.inuse    = tcpsck,,,,
+network.sockstat.udp.inuse    = udpsck,,,,
+network.sockstat.raw.inuse    = rawsck,,,,
+network.sockstat.frag.inuse   = ip-frag,,,,
```

Comment:
Preferably only one space between the widest metric name and the equal sign.

Reply:
Follow on patch fixes the output column alignment and the space between the metric name and the '=' has been push onto the branch.